build.gradle

```diff
@@ -37,4 +38,12 @@ dependencies {
     annotationProcessor('org.projectlombok:lombok:1.18.12')
     compile('org.springframework.boot:spring-boot-starter-hateoas')
     compile('org.simplejavamail:simple-java-mail:4.1.3')
+    compile('javax.xml.bind:jaxb-api:2.3.0')
```

Comment:
Please add in the PRs description the reasons for adding these dependencies.

Reply:
Remove. Not needed